Header header

CandidateHoleMsg[] candidateHoles

sensor_msgs/Image image
